### ISSUE TITLE
Ground Roll Forward @ Zero Throttle

### ISSUE THREAD

**Issue by GertBFrobe** [mixMugz/msfs_tbm930_project#9]
I remember this on the stock TBM when MSFS 2020 first came out. Getting it again with the mod installed.
Removed the mod, so back to stock, no zero throttle ground roll.

G3000 mod installed
HOTAS Warthog

**Comment by ReZZeR13** [comment]
Its rolling for me too....it didnt seem to do it in .8

**Comment by mixMugz** [comment]
Real aircraft roll at zero throttle. Because this in not real zero.

**Comment by guifarias31** [comment]
> Real aircraft roll at zero throttle. Because this in not real zero.

That's correct. Real TBM goes forward with throttle on idle, and they even control the speed on taxi using beta mode.

**Comment by mixMugz** [comment]
> That's correct. Real TBM goes forward with throttle on idle, and they even control the speed on taxi using beta mode.

Welcome back dude! I thinking about making beta mode. But without help is to hard for now

**Comment by guifarias31** [comment]
> > That's correct. Real TBM goes forward with throttle on idle, and they even control the speed on taxi using beta mode.
> 
> Welcome back dude! I thinking about making beta mode. But without help is to hard for now

That would be a great ideia, although I have zero knowledge and no time anymore, so I can't help.
But have a look on Paul's flight model mod, maybe adding some of the changes to your mod would be a nice ideia. 
https://flightsim.to/file/3266/daher-tbm-930-flight-model-mod
